src/metaxy/models/feature_spec.py

```diff
@@ -139,6 +140,17 @@ def table_name(self) -> str:
         return self.key.table_name
 
 
+class _CodeVersionDescriptor:
```

Comment:
is it intentiona tha this is an almost duplicate to the one in feature?

Reply:
@danielgafni wait - https://github.com/anam-org/metaxy/pull/115/files should fix that